.changeset/modern-kings-live.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react-brand': patch
+---
+
+Updates Testimonial component to align with 2023 design refactor.
```

Comment:
Just because that's an internal codename...

```suggestion
Updated Testimonial component visuals, and enables customization of the quote color.
```


Might also be nice to show an example of the quote fill prop and the `<em>` thing.